commands/new_function.go

```diff
@@ -25,6 +26,7 @@ func init() {
 	newFunctionCmd.Flags().StringVar(&lang, "lang", "", "Language or template to use")
 	newFunctionCmd.Flags().StringVar(&gateway, "gateway", defaultGateway,
 		"Gateway URL to store in YAML stack file")
+	newFunctionCmd.Flags().StringVar(&image, "image", "", "Name for docker image")
```

Comment:
While an interesting change, this shouldn't be part of the PR.

Reply:
@ericstoekl please see this feedback.

Reply:
I've got a new PR on the way for this feature, which addresses this. I'll close that when it's ready.